imdb.demo/src/main/java/com/springboot/imdb/demo/dao/MovieActorRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.springboot.imdb.demo.dao;
+
+import com.springboot.imdb.demo.entity.Actor;
+import com.springboot.imdb.demo.entity.Movie;
+import com.springboot.imdb.demo.entity.MovieActor;
+import com.springboot.imdb.demo.model.MovieActorIdentity;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+import java.util.Optional;
+
+
+public interface MovieActorRepository extends CrudRepository<MovieActor, MovieActorIdentity> {
+
+    @Query("SELECT m.actor FROM MovieActor m  WHERE m.movieId=(:movieId)")
```

Comment:
Use @Repository and remove @Query(), without writing actual queries these simple tasks can be achieved